services/datcore-adapter/tests/unit/test_route_datasets.py

```diff
@@ -0,0 +1,119 @@
+# pylint:disable=unused-variable
+# pylint:disable=unused-argument
+# pylint:disable=redefined-outer-name
+
+from collections import namedtuple
+from pathlib import Path
+from typing import Dict, List
+
+import httpx
+import pytest
+from fastapi_pagination import Page
+from pydantic import parse_obj_as
+from simcore_service_datcore_adapter.models.schemas.datasets import (
+    DatasetMetaData,
+    FileMetaData,
+)
+from starlette import status
+
+pytestmark = pytest.mark.asyncio
+
+ps_dataset = namedtuple("ps_dataset", "id,name")
```

Comment:
STYLE: ``namedtuple`` is a class factory, therefore ``ps_dataset`` is a class which we normally use camelcase for :-), i.e. ``PsDataset``?

Reply:
yes. anyway this was a relica from testing the pennsieve python client. removed.